sql/expression/function/registry.go

```diff
@@ -43,4 +43,8 @@ var Defaults = sql.Functions{
 	"coalesce":      sql.FunctionN(NewCoalesce),
 	"json_extract":  sql.FunctionN(NewJSONExtract),
 	"connection_id": sql.Function0(NewConnectionID),
+	"ln": 			 sql.Function1(NewLn),
```

Comment:
Do we need all these expressions (Ln, Log2, Log10...)? Maybe we can register all of them (sql functions) as a one `NewLog`, transform them and delegate to one `computeLog` function - wdyt?

Reply:
Sure we can pass the base as a parameter in NewLog and use it a  guide for the rest of the common methods. Will push a fix today